packages/vscode-extension/src/commonlib/m365CodeSpaceLogin.ts

```diff
@@ -19,6 +19,11 @@ import {
 import { getDefaultString } from "../utils/localizeUtils";
 import { UserCancelError } from "./codeFlowLogin";
 
+const CodeSpacesScopes = [
```

Comment:
`TeamsFxAadScopes` would be more accurate here, and the M365 local login and codespaces login can be unified in our extension as suggested from VS Code team. 

Further more, better to add comment for this secret scope. for example,

```typescript
// By using `VSCODE_CLIENT_ID:<aad-app-id>` in the `vscode.authentication` API, 
// we can override the scope with our own first-party AAD app. 
// Check https://www.eliostruyf.com/microsoft-authentication-provider-visual-studio-code/ for more details.
```

Reply:
Thanks for the advice. Done.